kafka/client_async.py

```diff
@@ -28,8 +28,6 @@
 from .metrics.stats import Avg, Count, Rate
 from .metrics.stats.rate import TimeUnit
 from .protocol.metadata import MetadataRequest
-from .protocol.produce import ProduceRequest
-from .vendor import socketpair
```

Comment:
This one should be left in, as it monkey-patches `socket.socketpair()`: https://github.com/dpkp/kafka-python/blob/ba7afd9bc9362055ec0bedcf53eb6f8909dc22d2/kafka/vendor/socketpair.py#L58

Rather than removing, can you add a comment right above it saying something like "Although this looks unused, it actually monkey-patches `socket.socketpair()` and should be left in as long as we're using `socket.socketpair()` in this file."

Reply:
Got it! Yep, I can make that change